larq/models_test.py

```diff
@@ -244,6 +246,33 @@ def test_subclass_model_summary(snapshot, capsys):
     snapshot.assert_match(captured.out)
 
 
+def test_saved_model_summary(capsys):
+    model = tf.keras.models.Sequential(
+        [
+            lq.layers.QuantConv2D(
+                filters=32,
+                kernel_size=(3, 3),
+                kernel_quantizer="ste_sign",
+                input_shape=(64, 64, 1),
+                padding="same",
+            ),
+            tf.keras.layers.GlobalAvgPool2D(),
+            tf.keras.layers.Dense(10, activation="softmax"),
+        ]
+    )
+    lq.models.summary(model)
+    summary = capsys.readouterr().out
+
+    # Save and reload
+    with TemporaryDirectory() as dir:
```

Comment:
You can use the [pytest `tempdir` fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture) directly, instead of creating it manually.

Reply:
Even better to use the more modern `tmp_path: Path` from pytest, right?